AwesomeProject/App.js

```diff
@@ -1,15 +1,16 @@
 import React from 'react';
-import { Component, StyleSheet, View, Text, ScrollView } from 'react-native';
+import { StyleSheet, View, Text, ScrollView } from 'react-native';
 
 const Header = () => {
```

Comment:
File name should be the same with component name. If it is App then it should be called as App.

Reply:
But for me better to create folder 'screens'  and inside of it create folder 'AboutMe' with file 'AboutMe'. where you paste this code from here

Reply:
AboutMe.screen.js

Reply:
screens/AboutMe:
AboutMe.screen.js
AboutMe.styles.js

Reply:
Fixed